packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -299,6 +303,8 @@ class UserAccountsDrawerHeader extends StatefulWidget {
     this.margin = const EdgeInsets.only(bottom: 8.0),
     this.currentAccountPicture,
     this.otherAccountsPictures,
+    this.currentAccountPictureSize = const Size.square(72.0),
+    this.otherAccountsPicturesSize = const Size.square(40.0),
```

Comment:
Was the default for otherAccountsPicturesSize 48px originally?  Looking at the deleted code above.  Or is it now affected differently by the padding?

Reply:
A padding of 8px has been moved above the SizedBox related to the otherAccountsPicturesSize.  So the actual size is the same.

Reply:
Ah got it.